Syntax corrections & typos (#128347)
Azure/azure-content: articles/service-bus-messaging/automate-update-messaging-units.md

    To learn more about how autoscale settings work, especially how it picks a profile or condition and evaluates multiple rules, see [Understand Autoscale settings](/azure/azure-monitor/autoscale/autoscale-understanding-settings).          

    > [!NOTE]

>     > - The metrics you review to make decisions on autoscaling may be 5-10 minutes old. When you are dealing with spiky workloads, we recommend that you have shorter durations for scaling up and longer durations for scaling down. As Service Bus Premium is charged per hour, scaling down quickly will not reduce the costs for that hour. Instead, it is recomended to give enough time to ensure the reduced workload is stable before scaling down to ensure that there are enough messaging units to process spiky workloads.

Reviewer: Edits for formatting, clarity, typo fix, and active voice, not passive voice.

```suggestion
    >
    > The metrics that you use to make autoscaling decisions might be 5-10 minutes old. When you have spiky workloads, use shorter durations for scaling up and longer durations for scaling down.
    >
    > Service Bus Premium is charged per hour, so scaling down quickly doesn't reduce costs for that hour. Instead, provide enough time to make sure the reduced workload is stable before scaling down and that enough messaging units exist to process spiky workloads.
```

    >

>     > When scaling down, set the threshold to less than half of the scale-up threshold. For instance, if the scale-up threshold is 80%, set the scale-down threshold to 30-35% (something below 40%) to prevent continuous scaling up and down.This will prevent autoscale to switch between scaling up and down continuously.

Reviewer: Simpler wording and remove redundant info.

```suggestion
    > For scale-down, set the threshold to less than half of the scale-up threshold. For example, if the scale-up threshold is 80%, set the scale-down threshold to 30-35%, or anything below 40%, to prevent continuous scale up and down.
```
